bin/compile

```diff
@@ -15,6 +14,9 @@ fi
 APP_BASE="$(cat "${ENV_DIR}/APP_BASE")"
 
 (
+    if [ -f "${BUILD_DIR}/yarn.lock" ] && [ ! -f "${ENV_DIR}/${APP_BASE}/yarn.lock" ]; then
```

Comment:
Is this right? I'm a bit confused about ENV_DIR vs BUILD_DIR vs APP_BASE in this.